src/UI/words.tsx

```diff
@@ -812,6 +812,7 @@ const dict = {
   "common.compileWidget.compiling": "Compiling",
   "common.compileWidget.compilationDisabled.hint":
     "The server_compile setting is disabled. You can enable it on the settings page under the configuration tab.",
+  "common.emptyColumnHeader": "Empty Column Header",
```

Comment:
Considering this is for accessiblity reasons, it should state what the collumn is about. 
```suggestion
  "common.emptyColumnHeader": "Column header for collapsible-expandable action",
```